package.json

```diff
@@ -12,16 +12,15 @@
   "devDependencies": {
     "@commitlint/cli": "^7.0.0",
     "@commitlint/config-conventional": "^7.0.0",
-    "@commitlint/config-lerna-scopes": "^7.0.0",
     "@commitlint/travis-cli": "^7.0.0",
     "@types/mocha": "^5.0.0",
     "coveralls": "^3.0.0",
     "cz-conventional-changelog": "^2.1.0",
     "husky": "^0.14.3",
-    "lerna": "^2.9.1"
+    "lerna": "^3.1.2"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap",
+    "postinstall": "lerna bootstrap --no-ci",
```

Comment:
For the record, lerna 3.x forces `npm ci` instead of `npm i` for CI envs. But `npm ci` fails as we don't check in package-lock.json files. It's not working to set the `ci` flag to `false` in `lerna.json` as the value is overridden.

Reply:
See https://github.com/lerna/lerna/issues/1449

Reply:
Cross-posting https://github.com/lerna/lerna/issues/1449#issuecomment-414750425

> Next release should fix this for you properly.

AFAICT, the next release has been already published, see https://github.com/lerna/lerna/releases/tag/v3.1.3
> Bug Fixes
>
> add: Avoid passing bad scope to pacote
> global-options: Move env defaults to command superclass

@raymondfeng Do you have bandwidth to update our scripts and remove `--no-ci` in favor of setting`ci: false` in `lerna.json`?